otsclient/args.py

```diff
@@ -170,6 +171,17 @@ def parse_ots_args(raw_args):
                               nargs='+',
                               help='Filename')
 
+    parser_stamp.add_argument("--timeout", type=int, default=5,
+                              help="Default maximum timeout for stamp request. "
+                                   "Default: %(default)d")
+
+    parser_stamp.add_argument("--m-of-n", type=str, default="2of2",
+                              help="Commitments are sent to n remote calendars,"
+                                   "in the event of timeout the timestamp is considered "
+                                   "done if at least m calendars replied. "
+                                   "Default: %(default)s")
```

Comment:
It'd be simpler to just call this option `-m`

Remember that the "n" part of it is redundant: you've already specified it by in how many calendars you've selected.